poll/public/html/poll_edit.html

```diff
@@ -0,0 +1,96 @@
+             <textarea class="input setting-input" name="feedback" id="poll-feedback-editor">{{feedback}}</textarea>
+           </div>
+            <span class="tip setting-help">
+                This text will be displayed for the user as some extra feedback after they have
+                submitted their response to the poll.
+            </span>
+        </li>
+        <li class="field comp-setting-entry is-set">
+            <div class="wrapper-comp-setting">
+                <label class="label setting-label poll-setting-label" for="poll-private-results">Private Results</label>
+                <select id="poll-private-results" class="input setting-input" name="private_results">
+                    <!-- So far as I can see, there's not a proper style for checkboxes. LTI module does it this way. -->
+                    <option value="true" {% if private_results %} selected{% endif %}>True</option>
+                    <option value="false" {% if not private_results %} selected{% endif %}>False</option>
+                </select>
+            </div>
+            <span class="tip setting-help">
+                If this is set True, don't display results of the poll to the user.
+            </span>
+        </li>
+        <li class="field comp-setting-entry is-set">
+            <div class="wrapper-comp-setting">
+                <label class="label setting-label poll-setting-label" for="poll-max-submissions">Maximum Submissions</label>
+                <input id="poll-max-submissions" type="number" min="0" step="1" value="{{ max_submissions }}"/>
+            </div>
+            <span class="tip setting-help">
+                Maximum number of times a user may submit a poll. <strong>Setting this to a value other than 1 will imply that
+                'Private Results' should be true.</strong> Setting it to 0 will allow infinite resubmissions.
+            </span>
+        </li>
+        <li class="field comp-setting-entry is-set">
+            <p>
+                <strong>Notes:</strong>
+                If you change an answer's text, all students who voted for that choice will have their votes updated to
+                the new text. You'll want to avoid changing an answer from something like 'True' to 'False', accordingly.
+                If you delete an answer, any votes for that answer will also be deleted. Students whose choices are deleted
+                may vote again, but will not lose course progress.
+            </p>
+            {%  if multiquestion %}
+                <p>
+                    Questions must be similarly cared for. If a question's text is changed, any votes for that question will remain.
+                    If a question is deleted, any student who previously took the survey will be permitted to retake it, but will not
+                    lose course progress.
+                </p>
+            {% endif %}
+        </li>
+        <li id="poll-answer-marker"></li>
```

Comment:
We will redesign this UI anyway in a separate PR, moving questions and answers to a different tab in the dialogue.  This redesign will make the problem go away.